charts/kube-prometheus-stack/values.yaml

```diff
@@ -1203,6 +1203,52 @@ kube-state-metrics:
   podSecurityPolicy:
     enabled: true
 
+## Blackbox Exporter configuration
+##
+blackboxExporter:
```

Comment:
Would it be possible to support `namespaceOverride` like other subcharts?

Thanks for this PR by the way, looking forward to see it merged!

Reply:
This will require added it to the subchart. Happy to do so assuming this is a blocker.